modules/cnv-subscribing-to-hco-catalog.adoc

```diff
@@ -12,28 +12,36 @@ namespace access to the {CNVProductName} Operators.
 
 .Prerequisites
 
-* `OperatorGroup` and `CatalogSource` Custom Resource Definition objects (CRDs),
-both created in the `openshift-cnv` namespace
+* Create a project called `openshift-cnv`.
 
 .Procedure
 
 . Open a browser window and navigate to the {product-title} web console.
 
 . Navigate in the web console to the *Operators → OperatorHub* page.
 
-. Scroll or type a keyword into the *Filter by keyword* box to find the KubeVirt
-HyperConverged Cluster Operator, then click on it.
+. Scroll or type a keyword into the *Filter by keyword* box to find the
+{CNVProductNameStart} Operator, then click on it.
 
 . Read the information about the Operator and click *Install*.
 
 . On the *Create Operator Subscription* page:
-.. Select one of the following:
+.. Select *A specific namespace on the cluster* from the *Installation Mode*
+list and choose the `openshift-cnv` namespace.
++
+[NOTE]
+====
 *** *All namespaces on the cluster (default)* installs the Operator in the default
 `openshift-operators` namespace to watch and be made available to all namespaces
 in the cluster. This option is not always available.
 *** *A specific namespace on the cluster* allows you to choose a specific, single
 namespace in which to install the Operator. The Operator will only watch and be
 made available for use in this single namespace.
+====
+.. Select the 2.1 *Update Channel* option.
+.. Select an *Approval Strategy* option:
+*** *Automatic* is recommended. If you select this option, {CNVProductName}
+automatically updates when a new z-stream release is available.
```

Comment:
I would also mention how it works with "Manual" option.

Reply:
My impression is that "Manual" is not actually supported. I'm waiting on @rthallisey 's response to an email from Steve G. It sounded like the way the options are displayed in the UI might be changed so that automatic is the default. I'll rephrase this when I hear more.

Reply:
We don't need to mention Manual, but rather something like "use the default option". 

`Use the default upgrade policy, *Automatic*`